### DISCUSSION TITLE
Reset Scene Graph

### DISCUSSION THREAD

**Discussion by gnansai** [google/model-viewer#1941] [Q&A]
Hi, 

The Material properties and Textures of Models can be changed using Scene Graph. 
 
Is there any way to reset the Model Viewer Scene Graph to default materials and textures without reloading the page?

**Comment by elalish** [comment]
No, it's up to you to save the state and reapply it if you want to revert. That's what we do in the editor.

**Comment by gnansai** [comment]
Thank you for the reply, I will check with the editor.
